server_src/migrations/0001_printers_settingsapperance_13072021.migration.js

```diff
@@ -0,0 +1,15 @@
+const Printer = require("../models/Printer");
+
+module.exports = class PrinterSchemaMigrator13072021 {
+  async up(schemaVersion) {
+    // Do the migration
+    // Change the schema version if we may
+
+    Printer.updateMany({}, { $rename: { settingsApperance: "settingsAppearance" } });
+    Printer.updateMany({}, { $rename: { apikey: "apiKey" } });
+  }
+
+  down(schemaVersion) {}
+};
+
+module.exports = PrinterSchemaMigrator13072021;
```

Comment:
Love this

Reply:
Just to check the client knows about the apiKey change?

Reply:
Note: this code isnt run yet. Its something to show "this is how it will be later". I'd be ok with purging it from canary if its better.

In future we will test the migration fully. If the client can be easily updated we should in such cases, if not we should convert it to old props. Got a very neat mapping function in the new code, will show you later. Its basically PrinterClean but then on-the-fly (no calculations, just flattening and mapping of properties to match the client interface)

Reply:
Nah it's fine to me if it's like that. 

The client thankfully is all in one place where it utilises those keys so it's not a hard update.